File: app/src/main/java/com/callmangement/imagepicker/ui/camera/CameraModule.java
package com.callmangement.imagepicker.ui.camera;

import android.content.Context;
import android.content.Intent;

import com.callmangement.imagepicker.model.Config;


/**
 * Created by hoanglam on 8/18/17.
 */

public interface CameraModule {
    Intent getCameraIntent(Context context, Config config);

    void getImage(Context context, Intent intent, OnImageReadyListener imageReadyListener);
}

File: app/src/main/java/com/callmangement/ehr/imagepicker/ui/camera/CameraModule.kt
package com.callmangement.ehr.imagepicker.ui.camera

import android.content.Context
import android.content.Intent
import com.callmangement.ehr.imagepicker.model.Config

/**
 * Created by hoanglam on 8/18/17.
 */
interface CameraModule {
    fun getCameraIntent(context: Context?, config: Config?): Intent?

    fun getImage(context: Context?, intent: Intent?, imageReadyListener: OnImageReadyListener?)
}
